internal/testhelpers/resources.go

```diff
@@ -271,6 +348,57 @@ func (cc *TestCaseClient) ExpectPodContainerCount(ctx context.Context, podSelect
 	return nil
 }
 
+// ExpectPodReady finds a deployment and keeps checking until the number of
+// containers on the deployment's PodSpec.Containers == count. Returns error after 30 seconds
+// if the containers do not match.
+func (cc *TestCaseClient) ExpectPodReady(ctx context.Context, podSelector *metav1.LabelSelector, allOrAny string) error {
+
+	var (
+		countBadPods int
+		countPods    int
+	)
+
+	err := RetryUntilSuccess(24, DefaultRetryInterval, func() error {
```

Comment:
This can just be `return RetryUntilSucess( /* ... */ )`.

Reply:
Fixed.